### PR TITLE
Update workflows

### LINKED ISSUE #68: Update workflow to add new badge and remove clutter.
**Steps**

- [x] Remove clutter from the workflow (I.e., unnecessary)
- [x] Add new badge that shows the total number of tests

**Story**

As a developer on this project, it is nice to have a simpler workflow that's easier to read and access. Additionally, have the ability to view the number of tests is very useful to see. This makes development much easier and it allows us to monitor the number of tests we have!

### PR DESCRIPTION
**Pull request by BerkayDur**
Update workflows to now be more succinct and readable. I've also added a new badge to see the total number of tests!!

Closes #68

### CODE REVIEW

File: .util/pylint_scores.txt
Comment (A-Faris) [inline .util/pylint_scores.txt:1]: Nice pylint score

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -103,6 +75,23 @@ jobs:
           CODE_QUALITY: ${{ env.PASSED_PERCENTAGE }}
           BADGE_PATH: .github/badges/passed_percentage.svg
 
+      # Get the number_of_tests from pytest_scores.json
+      - name: Obtain total_tests from JSON
+        if: '!cancelled()'
```
Comment (A-Faris) [inline .github/workflows/main.yaml:80]: Should all of these be removed?

File: .github/workflows/main.yaml
Comment (A-Faris) [inline .github/workflows/main.yaml:1]: File looks cleaner now

**Review by A-Faris — APPROVED** [review]
all looks good

File: .github/workflows/main.yaml (hunk 1 above)
Comment (BerkayDur) [reply .github/workflows/main.yaml:80]: Nope, this is used to continue execution even if pylint or pytest fails

**Pull request merged**